### PR TITLE
Update Conversion.vb to follow guidelines

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Conversion.vb [hunk 1]
```diff
@@ -56,19 +56,19 @@ Namespace Microsoft.VisualBasic
             Return Information.Err().Description
         End Function
 
-        Public Function ErrorToString(ByVal ErrorNumber As Integer) As String
-            If ErrorNumber >= MAX_ERR_NUMBER Then
+        Public Function ErrorToString(ByVal errorNumber As Integer) As String
```
Comment (stephentoub) [inline src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Conversion.vb]: This is a public API.  Changing the casing of an argument is a breaking change.

File: src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Conversion.vb [hunk 2]
```diff
@@ -79,50 +79,50 @@ Namespace Microsoft.VisualBasic
         ' Numeric functions.
         '============================================================================
 
-        Public Function Fix(ByVal Number As Short) As Short
-            Return Number
+        Public Function Fix(ByVal number As Short) As Short
```
Comment (stephentoub) [inline src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Conversion.vb]: Same for all of these.

**Comment by stephentoub** [comment]
@Youssef1313, thank you for the PR, but it's changing public API surface area.  Changing the casing of parameter names is a breaking change.

**Comment by Youssef1313** [comment]
> 
> 
> @Youssef1313, thank you for the PR, but it's changing public API surface area. Changing the casing of parameter names is a breaking change.

@stephentoub, Thanks for clarification. There was another change to use NameOf, should I open a new PR for it ?

**Comment by stephentoub** [comment]
> There were other changes to use NameOf, should I open a new PR for it ?

Yes, thank you.

**Pull request closed without merging**